pkg/image/qemu.go

```diff
@@ -121,7 +122,11 @@ func convertToRaw(src, dest string, preallocate bool) error {
 	}
 	if err != nil {
 		os.Remove(dest)
-		return errors.Wrap(err, "could not convert image to raw")
+		errorMsg := "could not convert image to raw"
+		if nbdkitLog, err := ioutil.ReadFile("/tmp/nbdkit.log"); err == nil {
```

Comment:
Can you create a constant for the log name?

Reply:
Done